src/components/common/ClientIntakeForm/IntakeForms/Finances.js

```diff
@@ -1,175 +1,189 @@
 import React from 'react';
-import { Form, Input, Checkbox } from 'antd';
+import { Form, Checkbox, Input } from 'antd';
+import { useState } from 'react';
+
+const initialFormValues = {
+  eviction: false,
+  landlordDebt: false,
+  criminalHistory: false,
+  poorCredit: false,
+  opName: false,
+  SSI: false,
+  SSDI: false,
+  studentLoan: false,
+  medicalBills: false,
+  creditCard: false,
+  autoLoan: false,
+};
+
+const familyHistory = [
+  {
+    name: 'eviction',
+    label: 'Eviction',
+  },
+  {
+    name: 'landlordDebt',
+    label: 'Landlord Debt',
+  },
+  {
+    name: 'criminalHistory',
+    label: 'Criminal History',
+  },
+  {
+    name: 'poorCredit',
```

Comment:
How come the name is poor credit and not no credit history? does it matter?

Reply:
good catch, i will fix this

Reply:
thank you!